src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -178,9 +181,7 @@ theorem getMsbD_eq_getMsb?_getD (x : BitVec w) (i : Nat) :
       intros
       omega
 
--- We choose `eq_of_getLsbD_eq` as the `@[ext]` theorem for `BitVec`
--- somewhat arbitrarily over `eq_of_getMsbD_eq`.
-@[ext] theorem eq_of_getLsbD_eq {x y : BitVec w}
```

Comment:
I think the ext annotation autogenerates a `BitVec.eq_of_getLsbD_eq_iff` declaration, which probably should be added explicitly. Not sure if there is a "right" way to autogenerate the iff versions of ext lemmas for types with multiple reasonable ext lemmas.

Reply:
I added this.